pallets/automation-time/src/lib.rs

```diff
@@ -208,6 +211,10 @@ pub mod pallet {
 		InvalidAmount,
 		/// Sender cannot transfer money to self.
 		TransferToSelf,
+		/// Insufficient balance to pay execution fee.
+		InsufficientBalance,
+		/// Account liquidity restrictions prevent withdrawal.
+		LiquidityRestrictions,
```

Comment:
we don't allow for liquidity in our chain tho?

Reply:
This is the error the Balance pallet will throw. I am just matching it.

The error occurs because you have enough funds, but the value of your funds after the action would be below what the withdrawal type allows for.